src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -107,11 +137,6 @@ public void parseAddress_null_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> ParserUtil.parseAddress((String) null));
     }
 
-    @Test
-    public void parseAddress_invalidValue_throwsParseException() {
-        assertThrows(ParseException.class, () -> ParserUtil.parseAddress(INVALID_ADDRESS));
-    }
-
```

Comment:
this was removed because I found it was impossible to test given the behavior of parseAddress to test; 
given that "" is now accepted, the invalid case would be " " or " [ADDRESS]", which is trimmed in the parseAddress method